images/linux/scripts/installers/java-tools.sh

```diff
@@ -45,8 +45,14 @@ enableRepositories() {
     fi
 
     # Add Addoptium PPA
-    wget -qO - "https://packages.adoptium.net/artifactory/api/gpg/key/public" | apt-key add -
-    add-apt-repository --yes https://packages.adoptium.net/artifactory/deb/
+    if isUbuntu22; then
+        # apt-key is deprecated on Ubuntu22, dearmor and add manually
+        wget -qO - https://packages.adoptium.net/artifactory/api/gpg/key/public | gpg --dearmor > /usr/share/keyrings/adoptium.gpg
+        echo "deb [signed-by=/usr/share/keyrings/adoptium.gpg] https://packages.adoptium.net/artifactory/deb/ jammy main" > /etc/apt/sources.list.d/adoptium.list
```

Comment:
We shouldn't hardcoded jammy.

Reply:
well, it is already called only under ubuntu22 so the distro name is predictable in any single way